axiom/tokens_integration_test.go

```diff
@@ -0,0 +1,96 @@
+package axiom_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/axiomhq/axiom-go/axiom"
+)
+
+// TokensTestSuite tests all methods of the Axiom Tokens API against a
+// live deployment.
+type TokensTestSuite struct {
+	IntegrationTestSuite
+
+	apiToken *axiom.APIToken
+}
+
+func TestTokensTestSuite(t *testing.T) {
+	suite.Run(t, new(TokensTestSuite))
+}
+
+func (s *TokensTestSuite) SetupSuite() {
+	s.IntegrationTestSuite.SetupSuite()
+}
+
+func (s *TokensTestSuite) TearDownSuite() {
+	s.IntegrationTestSuite.TearDownSuite()
+}
```

Comment:
No need to add that if you're not doing any custom setup in there. Since you're inheriting `IntegrationTestSuite`, this effectively just overrides `IntegrationTestSuite.SetupSuite` and `IntegrationTestSuite.TearDownSuite` just to call them from those method overrides.

Reply:
Thanks